meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Base image and common roles
   company: Stack Builders
   license: MIT
-  min_ansible_version: 2.4.0
+  min_ansible_version: 2.6.0
```

Comment:
@nickovivar I think you also need to get rid of the following files:
- https://github.com/stackbuilders/sb-debian-base/blob/master/requirements-2.4.txt
- https://github.com/stackbuilders/sb-debian-base/blob/master/requirements-2.5.txt

Reply:
@nickovivar could you also update the CHANGELOG? Additionally, is it a backward-incompatible change? Or it just produces a warning when trying to run this role with an old Ansible version

Reply:
@nickovivar I also found there is a section in the [README.md](https://github.com/stackbuilders/sb-debian-base/blob/master/README.md) file that talks about Ansible minimum version:

```diff
- Generic Debian image for servers. This Galaxy requires Ansible 2.4.0
+ Generic Debian image for servers. This Galaxy requires Ansible 2.6.0
```

Could you please update that section too?